admin_manual/release_notes.rst

```diff
@@ -11,6 +12,45 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.3_release_notes_label:
+
+Changes in 10.0.3
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues of ownCloud Server 10.0.3 that need your attention:
+
+**The full ownCloud Server 10.0.3 changelog can be found here: https://github.com/owncloud/core/blob/stable10/CHANGELOG.md**
+
+* It is now possible to directly upgrade from 8.2.11 to 10.0.3 in a single upgrade process.
+* Added occ command to list routes which can help administrators setting up network firewall rules.
+* 'occ upgrade' is now verbose by default.
+    * Administrators may need to adjust scripts for automated setup/upgrade procedures that rely on 'occ upgrade' outputs.
+
+* Reenabled medial search by default
+    * Enables partial search in sharing dialog autocompletion (e.g. a user wants to share with the user "Peter": Entering "pe" will find the user, entering "ter" will only find the user if the option is enabled)
+    * New default is set to enabled as there is no performance impact anymore due to the introduction of the user account table in ownCloud Server 10.0.1.
+    * Please check the setting. You need to disable it if the functionality is undesired.
```

Comment:
"You now need to disable it explicitly" maybe ?

Reply:
added